pages/common/luac.md

```diff
@@ -0,0 +1,11 @@
+# luac
+
+> Lua bytecode compiler.
+
+- Compile a Lua source file to Lua bytecode:
+
+`luac -o {{byte_code.luac}} {{source.lua}}`
+
+- Don't include debug information in the output:
```

Comment:
How about "Do not include debug symbols in the output:" ? So that we match the flag `-s` with a word in the description.